packages/server-dev/pages/api/page/[pageId].js

```diff
@@ -14,27 +14,19 @@
   limitations under the License.
 */
 
-import { createApiContext, getPageConfig } from '@lowdefy/api';
+import { getPageConfig } from '@lowdefy/api';
 
-import config from '../../../build/config.json';
-import fileCache from '../../../lib/fileCache.js';
-import getServerSession from '../../../lib/auth/getServerSession.js';
-
-export default async function handler(req, res) {
-  const session = await getServerSession({ req, res });
-  const apiContext = createApiContext({
-    buildDirectory: './build',
-    config,
-    fileCache,
-    logger: console,
-    session,
-  });
+import apiWrapper from '../../../lib/server/apiWrapper.js';
 
+async function handler({ context, req, res }) {
   const { pageId } = req.query;
-  const pageConfig = await getPageConfig(apiContext, { pageId });
+  const pageConfig = await getPageConfig(context, { pageId });
+  // TODO: Do we log here?
```

Comment:
I think yes. We'd want to log the page id that was requested also. This can be useful for debugging role based error, where you've made a change which prohibits users from accessing a page because of their role, so both role, requested page id and the fact that 404 was served is important.